chef-utils/spec/unit/dsl/architecture_spec.rb

```diff
@@ -131,6 +131,17 @@ def arch_reports_true_for(*args)
 
     arch_reports_true_for(:armhf?, :_32_bit?, :arm?)
   end
+  context "on armv6l" do
+    let(:arch) { "armhf" }
+
+    arch_reports_true_for(:armhf?, :_32_bit?, :arm?)
+  end
+  context "on armv7l" do
+    let(:arch) { "armhf" }
+
+    arch_reports_true_for(:armhf?, :_32_bit?, :arm?)
+  end
```

Comment:
I suspect these tests should have `let(:arch) { "armv6l" }`, etc.

Reply:
I opened https://github.com/chef/chef/pull/9892 to fix the tests